rust/src/treefile.rs

```diff
@@ -754,8 +754,13 @@ impl Treefile {
         self.parsed.base.selinux.unwrap_or(true)
     }
 
-    pub(crate) fn get_releasever(&self) -> &str {
-        self.parsed.base.releasever.as_deref().unwrap_or_default()
+    pub(crate) fn get_releasever(&self) -> String {
+        self.parsed
+            .base
+            .releasever
+            .as_ref()
+            .map(|rv| rv.clone())
```

Comment:
I think this is just [.cloned()](https://doc.rust-lang.org/std/option/enum.Option.html#method.cloned-1)